### DISCUSSION TITLE
Updating already deployed contract

### DISCUSSION THREAD

**Discussion by mpixelz** [hashlips-lab/nft-erc721-collection#319] [Q&A]
Hi,
We have already deployed a contract and have gone thru the whitelisted, pre-sale, reveal and public mint functions..
but as the market is in downtrend we would like to update the price of pre-sale and public mint.
so we can have a lower pre-sale and still get the option to update the price higher on public mint. i see we have an option to update the public mint price on etherscan contract but is there a way to update both thru the contract?
we're using v1 of the tool here.
Please let me know if that is possible.

Also,
If its a good idea to update to v2 of the contract? and if yes then how?
Thanks.

**Comment by ClubPEC** [comment]
Hi mpixelz

yes you can update all prices through your ERC721.

Just follow the usual process of closing & opening the sale. 

before opening any sale, just change the price, max mint etc in config.js and save it. the run.

Hope this helps 

Good luck

@club_pec
